packages/builder/src/components/common/CodeEditor/CodeEditor.svelte

```diff
@@ -428,6 +477,50 @@
   <div tabindex="-1" bind:this={textarea} />
 </div>
 
+{#if aiGenEnabled}
+  <button
```

Comment:
Is there a reason on why not using bbui buttons? Are the designs not matching?

Reply:
This started life as a <div>, but that caused a lint error about divs not being keyboard accessible. I didn't know how to attach a class to a <Button> so I went with <button>.

It is highly likely this will change before we ship this, my plan was to ask Cheeks or Peter to have a once over and make it look nice.